### PR TITLE
Log warning if `onMessageDeleted` hook passed to `subscribeToRoom`

### PR DESCRIPTION
**Pull request by minaorangina**
V2 does not send message events. Adding a warning to customers can know what to expect

### CODE REVIEW

**Comment by minaorangina** [comment]
Build seems to be failing for unrelated reasons

File: src/current-user.js [hunk 1]
```diff
@@ -434,6 +434,13 @@ export class CurrentUser {
   subscribeToRoom({ roomId, hooks = {}, messageLimit, serverInstance } = {}) {
     typeCheck("roomId", "string", roomId)
     typeCheckObj("hooks", "function", hooks)
+    if (!serverInstance && hooks.onMessageDeleted) {
+      // v2 does not send message_deleted events
+      // eslint-disable-next-line no-console
+      console.warn(
```
Comment (callum-oakley) [inline src/current-user.js]: We have access to `this.logger` so no need to use `console`
```suggestion
      this.logger.warn(
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ This project adheres to [Semantic Versioning Scheme](http://semver.org)
 
 ## [Unreleased](https://github.com/pusher/chatkit-client-js/compare/1.13.2...HEAD)
 
+### Changes
+- Logs a warning if `onMessageReceived` hook is passed to `subscribeToRoom`
```
Comment (callum-oakley) [inline CHANGELOG.md]:
```suggestion
- Logs a warning if `onMessageDeleted` hook is passed to `subscribeToRoom`
```

File: src/current-user.js [hunk 3]
```diff
@@ -434,6 +434,13 @@ export class CurrentUser {
   subscribeToRoom({ roomId, hooks = {}, messageLimit, serverInstance } = {}) {
     typeCheck("roomId", "string", roomId)
     typeCheckObj("hooks", "function", hooks)
+    if (!serverInstance && hooks.onMessageDeleted) {
+      // v2 does not send message_deleted events
+      // eslint-disable-next-line no-console
```
Comment (callum-oakley) [inline src/current-user.js]: don't need the eslint-disable any more
```suggestion
```

**Pull request merged**